proposals/0000-template.rst

```diff
@@ -100,6 +100,23 @@ Hopefully this section will be empty by the time the proposal is brought to
 the steering committee.
 
 
+Proposal Collaboration
```

Comment:
This is duplicated from above?

Reply:
Yes, because we have two different templates which should remain in sync.